weback_unofficial/thermostat.py

```diff
@@ -0,0 +1,62 @@
+from weback_unofficial.client import WebackApi, BaseDevice
+
+# uses a weekly plan (Mon, Tues, Wed, Thur, Fri, Sat, Sun)
+AUTO = 'auto'
+# uses the temperature you set in the set_temp property
+MANUAL = 'hand'
+
+WORKING_MODES = {AUTO, MANUAL}
+TEMP_MULTIPLIER = 2
```

Comment:
You need to describe this constant in comments (what is meant for and why it's 2)